src/openrct2/config/Config.h

```diff
@@ -85,6 +85,10 @@ namespace OpenRCT2::Config
         bool InvertViewportDrag;
         bool ZoomToCursor;
 
+        // Gamepad
+        int32_t GamepadDeadzone;
+        float GamepadSensitivity;
```

Comment:
Let’s make these adhere to the new rules straight away:

```suggestion
        int32_t gamepadDeadzone;
        float gamepadSensitivity;
```